clients/unshaded/src/main/java/tachyon/client/FileInStream.java

```diff
@@ -153,7 +156,7 @@ public void seek(long pos) throws IOException {
     if (pos < 0) {
       throw new IOException("Seek position is negative: " + pos);
     } else if (pos > mFileLength) {
-      throw new IOException("Seek position is past EOF: " + pos + ", fileSize = " + mFileLength);
+      throw new EOFException("Seek position is past EOF: " + pos + ", fileSize = " + mFileLength);
```

Comment:
Why is the exception message style inconsistent? I think it would be best to stick with the current style in the code base (not using String.format).